components/python/pygobject-3-legacy/pygobject-3-27.p5m

```diff
@@ -0,0 +1,69 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Gary Mills
+# Copyright 2016 Alexander Pyhalov
+#
+
+# This file intentionally hardcodes Python 2.7 instead of using the PYVER
+# subsititutions to avoid triggering the auto-generation of a generic
+# library/python/pygobject-3 in this component, since that's being done in
+# ../pygobject-3 instead to cover the other python 3.x versions correctly.
+
+set name=pkg.fmri value=pkg:/library/python/pygobject-3-27@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="Python bindings for GObject"
+set name=info.classification value="org.opensolaris.category.2008:Development/Python"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license COPYING license="LGPL v2.1"
+
+depend type=require fmri=runtime/python-27
+depend type=require fmri=library/python/pygobject-3
+depend type=require fmri=library/desktop/gtk3@3.14.15,5.11-2016.0.0.2
```

Comment:
I am not sure whether it's a good idea to depend on exact versions. I recommend to use
depend type=require fmri=library/desktop/gtk3
instead.

Reply:
I noticed that version string in the original manifest.  I don't know why it depended on an obsolete version, but I decided not to alter it.  Certainly, omitting the version string is better.  That change will appear in the next push.